### DISCUSSION TITLE
Credits consumed before even deploying

### DISCUSSION THREAD

**Discussion by wigsaparelli** [serverless/serverless#12575] [General]
I'm just getting started with serverless framework v4 CLI and using my 2 free instance credits.
I notice my credits are consumed even if I just run `serverless print` or `serverless package` which don't even deploy anything. The only way I've found to release the credits is to run `serverless remove` which **throws an error** because there is nothing to remove(!) but at least it releases a credit.  Am I doing something wrong as this doesn't seem right.

**Comment by austencollins** [comment]
Hi @wigsaparelli, We're currently refining our approach and have decided to track everything for now given the breadth of ways the Framework is used. As you discovered, you can remove it, and we'll make this experience better to confirm removal in the CLI. Our main focus ATM is improving the visibility of tracked Instances in the Serverless Framework Dashboard. This ensures transparency for our customers and will help us understand the best approach. We're a couple weeks away from releasing this, as it's almost done.

**Comment by levi-20** [comment]
Hi @austencollins ,
is the credit count in local CLI removal released or yet to be? I'm using 4.4.16 it still counts a credit for every service I run `serverless package` .
